tasks/trimtrailingspaces.js

```diff
@@ -41,21 +44,25 @@ module.exports = function trimtrailingspaces(grunt) {
         var trimmed = content.replace(/[ \f\t\v]*$/gm, '');
 
         if (content !== trimmed) {
-          grunt.verbose.writeln('Needed trimming, file: ' + src);
-          ++changedFileCount;
+          grunt.log.ok('Needed trimming, file: ' + src);
           grunt.file.write(destination, trimmed, fsOptions);
+          changedFileCount++;
         }
         else if (src !== destination) {
           // Copy to destination if not in place modification
           grunt.file.copy(src, destination, fsOptions);
         }
-
-
       });
     });
 
-    if (changedFileCount > 0 && options.failIfTrimmed) {
-      grunt.fail.warn(changedFileCount + ' files had whitespace trimmed, and the failIfTrimmed option is set to true.', 6);
+    successCount = fileCount - changedFileCount;
+    grunt.log.ok(successCount + ' file' + (successCount === 1 ? '' : 's') + ' free of trailing whitespace.');
+
+    if (changedFileCount) {
```

Comment:
Please use `changedFileCount > 0` as the given variable should be a number always.
In case it would be used as the diff suggest, the variable should be a boolean.